blocks/related-content-content-source-block/sources/related-content.js

```diff
@@ -22,7 +22,7 @@ export default {
   transform: (data, query) => {
     if (data && data.basic && typeof Array.isArray(data.basic)) {
       return {
-        basic: data.basic.map((ele) => (
+        content_elements: data.basic.map((ele) => (
```

Comment:
Could we possibly destructure this to `contentElements` or is `content_elements` a requirement?

Reply:
is a requeriment, is the top level element used by ANS collections.